src/tests/suite-ui/omegaEdit-driver.test.ts

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
My main comment for this file is that I would like to see more whitespaces. I feel as those some of these lines can be broken up more

Reply:
As you wish.